src/Common/MorganStanley.ComposeUI.Common.VisualUtils/MorganStanley.ComposeUI.Common.VisualUtils.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<!--
+********************************************************************************************************
+
+Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License").
+You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+Unless required by applicable law or agreed to in writing, software distributed under the License
+is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and limitations under the License.
+
+********************************************************************************************************
+-->
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="NP.Avalonia.Visuals" Version="1.0.24" />
```

Comment:
What does this package do?

Reply:
This package replaces 3rd party tooling and has a lot of useful controls, behaviors and converters. It is documented extensively on the codeproject and UniDock package that we use for docking depends on it.

Reply:
The VisualUtils will be renamed to underscore that it is an Avalonia implementation.

Reply:
renamed it to MorganStanley.ComposeUI.Avalonia.VisualUtils and placed it within Visuals/Avalonia physical and solution folders.